lib/opensusebasetest.pm

```diff
@@ -483,6 +483,7 @@ sub wait_boot {
             type_line_svirt '', expect => $login_ready, timeout => $ready_time + 100, fail_message => 'Could not find login prompt';
             type_line_svirt "root", expect => 'Password';
             type_line_svirt "$testapi::password";
+            sleep 10;
```

Comment:
Please add comments here to describe why need add 10s sleep.

Reply:
added, thanks.

Reply:
After looking at the above again I wonder how that should help. Why call `sleep 10` *after* typing the password?